client/gutenberg/extensions/simple-payments/editor.js

```diff
@@ -26,7 +25,8 @@ registerBlockType( 'jetpack/simple-payments', {
 		'Lets you create and embed credit and debit card payment buttons with minimal setup.'
 	),
 
-	icon: <GridiconMoney />,
+	icon:
+		'<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill="none" d="M0 0h24v24H0V0z"/><path d="M20 4H4c-1.11 0-1.99.89-1.99 2L2 18c0 1.11.89 2 2 2h16c1.11 0 2-.89 2-2V6c0-1.11-.89-2-2-2zm0 14H4v-6h16v6zm0-10H4V6h16v2z"/></svg>',
```

Comment:
To keep things consistent, let's use jsx here. See e.g.:

https://github.com/Automattic/wp-calypso/blob/68cd492e4d1ba8cce6891e2a401060142c312786/client/gutenberg/extensions/markdown/editor.js#L30-L44

Reply:
It would be good to start using Gutenberg's `SVG`, `Path` and `G`:

https://github.com/WordPress/gutenberg/tree/master/packages/components/src/primitives/svg

```jsx
import { G, Path, SVG } from '@wordpress/components';

const MyIcon = () => (
	<SVG
		viewBox="0 0 24 24"
		xmlns="http://www.w3.org/2000/svg"
	>
		<Path fill="none" d="M0 0h24v24H0V0z" />
		<G>
			<Path d="M20 4v12H8V4h12m0-2H8L6 4v12l2 2h12l2-2V4l-2-2z" />
			<Path d="M12 12l1 2 3-3 3 4H9z" />
			<Path d="M2 6v14l2 2h14v-2H4V6H2z" />
		</G>
	</SVG>
);
```